go.mod

```diff
@@ -252,7 +252,8 @@ require (
 	github.com/xdg-go/stringprep v1.0.4 // indirect
 	github.com/youmark/pkcs8 v0.0.0-20181117223130-1be2e3e5546d // indirect
 	go.opencensus.io v0.24.0 // indirect
-	go.opentelemetry.io/otel/metric v1.20.0 // indirect
+	go.opentelemetry.io/otel/metric v1.28.0 // indirect
+	go.opentelemetry.io/otel/sdk v1.28.0
```

Comment:
I only meant to add this package go.opentelemetry.io/otel/sdk, not sure why the others were auto-updated?

Reply:
@mtoffl01 Try importing `go.opentelemetry.io/otel/sdk` in the same version when running `go get`, like this: `go get go.opentelemetry.io/otel/sdk@v1.20.0`.

You may want to revert the changes in `go.mod` and `go.sum` before running the command again.